index.html

```diff
@@ -27,6 +27,8 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <!-- Adding some new improvements to navigation! -->
```

Comment:
The enlargement and placement of the search bar with the navigation button next to each other at the top of the page have made it easier to see and use. Requests have been made to make this standard on all pages.